service-worker/worker/src/e2e/page-object.ts

```diff
@@ -0,0 +1,61 @@
+declare var browser;
+declare var element;
+declare var by;
+
+export class HarnessPageObject {
+  
+  sendKeysSlow(el, keys) {
+    keys.split('').forEach(char => el.sendKeys(char));
```

Comment:
Not sure - it's a solution I found online. Protractor/Chrome were dropping keys otherwise.